crates/task-impls/src/request.rs

```diff
@@ -236,7 +238,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> DelayedRequester<TYPES, I> {
     async fn handle_response_message(&self, message: SequencingMessage<TYPES>) {
```

Comment:
Am I understanding correctly that the request task also handles the response, i.e., if we don't have a response task, this request task can still do the handling in the `handle_response_message` function?

Would it be better to rename this task to something like `RequestResponse` to make it explicit that it's handling both requests and responses?

Reply:
Hmm I agree with the naming being confusing.  The problem is what I called the `Request` task is responsible for sending out requests for data that the node wants, and then getting the response.

What I call the `Response` task, is responsible for for taking incoming requests, creating a response and responding to the node asking for the data.

So in a way both tasks to request and response.  Would it make sense to call the `Request` task the `Requester` task, and the `Response` task the `Responder` task or something of that nature?

Reply:
Oh, I missed your reply before giving the PR approval. This makes sense. It's probably not necessary to do the renaming. I'll leave it to your discretion then!